src/_P036_FrameOLED.ino

```diff
@@ -391,12 +480,25 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
 
     case PLUGIN_WEBFORM_SAVE:
       {
+        if (nullptr == P036_data) {
+#ifdef PLUGIN_036_DEBUG
+          addLog(LOG_LEVEL_INFO, F("P036_PLUGIN_WEBFORM_SAVE NoData -> Create temp ..."));
+#endif  // PLUGIN_036_DEBUG
+          P036_data_temp = new P036_data_struct();  // temporary for SaveCustomTaskSettings()
+        }
+        else {
+#ifdef PLUGIN_036_DEBUG
+          addLog(LOG_LEVEL_INFO, F("P036_PLUGIN_WEBFORM_SAVE ..."));
+#endif  // PLUGIN_036_DEBUG
+          P036_data_temp = P036_data;             // updating initialized values
+        }
```

Comment:
This section I don't understand.
If the `P036_data` is not available, then you create one... that part I do (somewhat) understand.
But why do you copy the pointer to P036_data_temp?

This makes the administration of the life time of the data object next to impossible.
If there is only a single owner of the pointer, then you know you have to delete the object before re-assigning a new one (or setting it to `nullptr`)
But if there are other pointers out there pointing to the same object, then you can't be sure if someone is de-referencing it after the object has been destroyed.
Or it is never deleted and then we're having a memory leak.  (or 2 versions of the same data)

Reply:
Changed, not using a temporary pointer in `PLUGIN_WEBFORM_SAVE` anymore.
The original idea was to update the current values if the task is already initialized.
Calling `SaveCustomTaskSettings()` was coded with the pointer -> changed.